todo_app/data/mongodb.py

```diff
@@ -0,0 +1,92 @@
+import os
+from enum import Enum
+from datetime import datetime
+from typing import List
+
+import pymongo
+from bson.objectid import ObjectId
+
+from todo_app.data.item import Item
+
+
+class CardStatus(Enum):
```

Comment:
The fact this has transferred almost identically over from the Trello file suggests we've done a good job at separating it from the underlying storage implementation, which might suggest we should pull it out into a separate file to acknowledge that intention